### PR TITLE
Update to 3.12

### PR DESCRIPTION
**Pull request by vasind**
1. Update to Ember 3.12.0 using ember-cli-update
2. Drop Node support for 8
3. Remove sendAction deprecations

### CODE REVIEW

**Comment by rwjblue** [comment]
Curious, why 3.12?

**Comment by vasind** [comment]
Just a baby step towards pre-Octane. Any suggestions please @rwjblue

File: LICENSE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014 Yapp, Inc and contributors
+Copyright (c) 2019 Yapp, Inc and contributors
```
Comment (Alonski) [inline LICENSE.md:1]: Maybe we can just remove this Copyright instead of updating it?
@lukemelia Do you mind? What do you think?

Comment (lukemelia) [reply LICENSE.md:1]: @Alonski no objection here

Comment (Alonski) [reply LICENSE.md:1]: @vasind Can you do this tiny thing :)

**Pull request merged**